tests/director/tests/main_test.go

```diff
@@ -27,11 +35,18 @@ func TestMain(m *testing.M) {
 
 	testctx.Tc, err = testctx.NewTestContext()
```

Comment:
As far as I remember we discussed, in the PR with the new structure of the e2e tests, that this assignment is unnecessary as test.Tc is initialized in the init method of the package. Could we remove it?